devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/extension/QuarkusPluginExtension.java

```diff
@@ -31,23 +31,38 @@
 import io.quarkus.runtime.LaunchMode;
 
 public class QuarkusPluginExtension {
-
     private final Project project;
 
-    private File outputDirectory;
-
-    private String finalName;
-
-    private File sourceDir;
-
-    private File workingDir;
-
-    private File outputConfigDirectory;
+    private final Property<String> finalName;
 
+    //    private final DirectoryProperty sourceDirectory;
+    //    private final DirectoryProperty outputDirectory;
+    //    private final DirectoryProperty outputConfigDirectory;
+    //    private final DirectoryProperty workingDirectory;
+    //
```

Comment:
@sebersole do we need these comments? And those below?

Reply:
Probably not  :)

Reply:
Speaking of... IJ says `io.quarkus.gradle.extension.QuarkusPluginExtension#appJarOrClasses` is never used though I did not trust it since it seems like not all modules (integration tests e.g.) do not get "imported".

Just mentioning it